src/libs/conduit/conduit_node.cpp

```diff
@@ -13501,6 +13505,200 @@ Node::find_first_data_ptr() const
     return res;
 }
 
+//---------------------------------------------------------------------------//
+bool
+Node::equals(const Node &n) const
+{
+    Node info;
+    return !diff(n, info);
```

Comment:
since the messages created by `diff` can include large strings when there are differences and we won't even to use all that info, we may want to use `ndiff`